explorer/src/components/SearchBar.tsx

```diff
@@ -189,6 +189,32 @@ function buildOptions(search: string, cluster: Cluster) {
     options.push(tokenOptions);
   }
 
+  if (!isNaN(Number(search))) {
+    options.push({
+      label: "Slot",
+      options: [
+        {
+          label: `Slot #${search}`,
+          value: search,
+          pathname: `/slot/${search}`,
+        },
+      ],
+    });
+  }
+
+  if (search.length > 42 && search.length < 45) {
+    options.push({
+      label: "Blockhash",
+      options: [
+        {
+          label: `Blockhash ${search}`,
+          value: search,
+          pathname: `/blockhash/${search}`,
+        },
+      ],
+    });
+  }
```

Comment:
Blockhash's look just like accounts actually. They are both base58 encoded 32 byte values. So this option should be added whenever the account option is added below

Reply:
Okay perfect, thank you!
Just updated that.